<launch>
	<arg name="marker_size" default="3.3" />
	<arg name="max_new_marker_error" default="0.05" />
	<arg name="max_track_error" default="0.2" />

	<!--arg name="cam_image_topic" default="/hrl_feeding_task/camera/depth_registered_points" /-->
	<arg name="cam_image_topic" default="/head_mount_kinect/depth_registered/points" />
	<arg name="cam_info_topic" default="/head_mount_kinect/rgb/camera_info" />		
	<arg name="output_frame" default="/torso_lift_link" />        

        <rosparam command="load" file="$(find hrl_multimodal_anomaly_detection)/params/anomaly_detection.yaml" />

        <!--node name="pcl_filter" pkg="hrl_multimodal_anomaly_detection" type="PCLFilter" respawn="false" output="screen" /-->

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>
